inc/object.class.php

```diff
@@ -600,6 +600,7 @@ function displayField($canedit, $name, $value, $template, $description = []) {
          $column_type = preg_replace('/[ (].+$/', '', $description['Type']);
```

Comment:
Problem was located in the regex.
```suggestion
         $column_type = preg_replace('/^([a-z]+)([ (].+)*$/', '$1', $description['Type']);
```

Reply:
done